### ISSUE TITLE
README

### ISSUE THREAD

**Issue by yerbn0409** [yerbn0409/movie_bigcon#1]
Overview : This program is designed to predict audience of Nut Job2 and Nam Han San Sung(남한산성, It's a korean movie) at 8days after opening. 
At first this program collects movie data(name of director and actors, movie code(used as primary key), movie name, name of movie distributor, movie scores and audience at 8days after opening. 
And then find correlation between movie data and audience so that learns with Deep Neural Network making weight matrix.
Now we can predict audience of Nut Job2 and Nam Han San Sung by using learned network.

movie0.py : It gets movie data(about 15,000 movies from 2004~2016) from API of Korean Film Council and makes it into pandas dataframe and then save as "boxoffice.csv". <- it's about training set
movie0_d8.py : It gets movie data(about Nut Job2 and Nam Han San Sung) from API of Korean Film Council and makes it into pandas dataframe and then save as "boxoffice_d8.csv". (d8 means day after 8) <- it's about test set

movie1.py : It reads boxoffice.csv, which made from movie0.py, and then gets more detailed information(seats, audience(day by day)...) about movies listed in boxoffice.csv, and merge each other so that save merged one as "total_movie.csv".
movie1_d8.py : It does same procedure on Nut Job2 and Nam Han San Sung as movie1.py

movie2.py : It gets movie star score(netizen evaluated and gave scores on movies, this program gets information about that) and made dataframe and then save it as "star_score.csv".

DNN.py : Using collected data(total_movie.csv and star_score.csv), it fits the neural network and predict audience. Note that there are not only continuous features but also categorical features. We treat them differently.
